snapfaas/bins/multivm/main.rs

```diff
@@ -13,14 +13,17 @@ use clap::{App, Arg};
 use log::{warn, info};
```

Comment:
So yeah, I believe you address this in your comment in the discussion, but it seems that this really should be two separate programs (perhaps even broken out to different crates): 1 program is the scheduler/external gateway that manages a cluster of faasten machines, while another program serves the role `multivm` has so far of managing the VMs on a single machine. Ideally, I think, the scheduler/gateway would be able to run continuously as nodes are added/removed (e.g. as they connect/disconnect) and work with, essentially, a dynamic number of workers, while a local node is clearly bound by the local resources allocated to it (e.g. the vailable memory and CPU on the machine, etc).

Reply:
That is the plan. I put them all together in `multivm` because I don't want to downgrade `multivm`, but the scheduler in the current state can work with a dynamic number of workers. It's easy enough to split it into two programs. However, since faasten only has a local store, having more than one worker machine leads to undefined behaviors. Also, if I need to make it a different crate, where should I put it? Under the root dir?

Reply:
Do you mind separating the two out? I feel the goal will be 99% completed if we do not. You can just add another binary  `sched` under `bins`. It does not need to be a separate crate. A crate can multiple binaries.